authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerHiveIT.java

```diff
@@ -1107,6 +1154,17 @@ public void testOnGrantedRolesToUser() {
         rangerAuthHivePlugin.onGrantedRolesToUser(Lists.newArrayList(role), userEntity1));
     verifyRoleInRanger(rangerAuthHivePlugin, role, Lists.newArrayList(userName1));
 
+    // test to throw an exception
+    RangerClientExtension client = Mockito.mock(RangerClientExtension.class);
+    RangerClientExtension originClient = rangerAuthHivePlugin.getRangerClient();
+    rangerAuthHivePlugin.setRangerClient(client);
+    when(client.searchUser(any())).thenReturn(Mockito.mock(VXUserList.class));
+    when(client.grantRole(any(), any())).thenThrow(new AuthorizationPluginException("test"));
```

Comment:
Is it `AuthorizationPluginException` or `RangerServiceException`?
It's a problem elsewhere.

Reply:
I correct this.